src/lib/features/listening-history/index.ts

```diff
@@ -1 +1,4 @@
-export { addToListeningHistory, listeningHistory } from './model/store';
+export {
+  addToLocalListeningHistory as addToListeningHistory,
```

Comment:
question: why do we only expose the "local" version of this function? won't this lead to unsynchronization?